app/src/main/java/com/android/unio/ui/home/Home.kt

```diff
@@ -102,152 +90,199 @@ fun HomeScreen(
+@Composable
+fun TopBar(
+    navigationAction: NavigationAction,
+    searchState: SearchViewModel.Status,
+    searchResults: List<Event>,
+    userViewModel: UserViewModel,
+    searchViewModel: SearchViewModel,
+    eventViewModel: EventViewModel,
+    paddingValues: PaddingValues
+) {
+  val nbOfTabs = 2
+  val context = LocalContext.current
+
+  Column(
+      horizontalAlignment = CenterHorizontally,
+      modifier =
+          Modifier.padding(paddingValues).padding(top = 20.dp, bottom = 50.dp).fillMaxSize()) {
+        var searchQuery by remember { mutableStateOf("") }
 
-          DockedSearchBar(
-              inputField = {
-                SearchBarDefaults.InputField(
-                    modifier = Modifier.testTag(HomeTestTags.SEARCH_BAR_INPUT),
-                    query = searchQuery,
-                    onQueryChange = {
-                      searchQuery = it
-                      searchViewModel.debouncedSearch(it, SearchViewModel.SearchType.EVENT)
-                    },
-                    onSearch = {},
-                    expanded = false,
-                    onExpandedChange = {},
-                    placeholder = {
-                      Text(text = context.getString(R.string.explore_search_placeholder))
-                    },
-                    trailingIcon = {
-                      if (searchState == SearchViewModel.Status.LOADING) {
-                        CircularProgressIndicator()
-                      } else {
-                        Icon(
-                            Icons.Default.Search,
-                            contentDescription =
-                                context.getString(R.string.home_content_description_search_icon))
-                      }
-                    },
-                )
-              },
-              expanded = false,
-              onExpandedChange = {},
-              modifier = Modifier.padding(horizontal = 16.dp).testTag(HomeTestTags.SEARCH_BAR)) {}
+        val list =
+            listOf(
+                context.getString(R.string.home_tab_all),
+                context.getString(R.string.home_tab_following))
+        val scope = rememberCoroutineScope()
+        val pagerState = rememberPagerState(initialPage = 0) { nbOfTabs }
 
-          // Event List
-          if (searchQuery.isNotEmpty() &&
-              (searchState == SearchViewModel.Status.SUCCESS ||
-                  searchState == SearchViewModel.Status.LOADING)) {
-            if (searchResults.isEmpty()) {
+        val sizeList = remember { mutableStateMapOf<Int, Pair<Float, Float>>() }
+
+        val progressFromFirstPage by remember {
+          derivedStateOf { pagerState.currentPageOffsetFraction + pagerState.currentPage.dp.value }
+        }
+        val colorScheme = MaterialTheme.colorScheme
+
+        // Tab Menu at the top of the page
+        TabRow(
+            selectedTabIndex = pagerState.currentPage,
+            contentColor = Color(0xFF362C28),
```

Comment:
Hard-coded color ? Won't we have issues with this when changing the app's color theme ?